File: 02.CSharp-Fundamentals/03.Arrays/Lab/DayOfWeek/DayOfWeek.cs
using System;

namespace DayOfWeek
{
    class DayOfWeek
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());

            string[] daysOfWeek = 
            {
                "Monday",
                "Tuesday",
                "Wednesday",
                "Thursday",
                "Friday",
                "Saturday",
                "Sunday",
            };

            if (n >= 1 && n <= 7)
            {
                Console.WriteLine(daysOfWeek[n - 1]);
            }
            else
            {
                Console.WriteLine("Invalid day!");
            }
        }
    }
}

File: 09.JS-Advanced/01.Syntax-Functions-and-Statements/Lab/dayOfWeek.js
function solve(dayOfWeek) {
    let dayAsString = String(dayOfWeek);

    switch (dayAsString.toLowerCase()) {
        case 'monday':
            return 1;

        case 'tuesday':
            return 2;

        case 'wednesday':
            return 3;

        case 'thursday':
            return 4;

        case 'friday':
            return 5;

        case 'saturday':
            return 6;

        case 'sunday':
            return 7;

        default:
            return 'error';
    }
}

console.log(solve('Monday'));
console.log(solve('Friday'));
console.log(solve('Invalid'));
